docs/explanation/measures.md

```diff
@@ -0,0 +1,299 @@
+atorva_80 | 2022-06-01 | 2022-06-30 | 0.5 | 5 | 10 | female
+
+
+Here the **`measure`** column always has the same value (`atorva_80`) because we only have a single measure defined, but if we had defined multiple measures then this would tell us which measure each row relates to.
+
+The **`interval_start`** and **`interval_end`** columns show the date range (inclusive) covered by each row. In this case there are 12 rows because each of the six months appears twice: once where `sex=male` and again where `sex=female`.
+
+The **`denominator`** column gives the number of patients for each interval-sex combination which match our denominator definition. The **`numerator`** column gives the number of patients which _also_ match the numerator definition. That is, a patient must match both the denominator and numerator definitions to be included in the numerator.
+
+The **`ratio`** is simply `numerator` divided by `denominator`.
+
+The **`sex`** column gives the value of the expression we supplied when defining `group_by`. Had we defined any other groupings (say `age_band`) then they would appear as additional columns here. 
+
+
+## Core concepts
+
+### Defining a measure
+
+A measure definition always starts by creating a measures collection object:
+```python
+measures = create_measures()
+```
+
+Each individual measure is then defined by calling [`measures.define_measure()`](../../reference/language/#Measures.define_measure):
+```python
+measures.define_measure(
+    name="atorva_80",
+    denominator=atorvastatin_rx.exists_for_patient() & has_recorded_sex,
+    numerator=atorvastatin_80_rx.exists_for_patient(),
+    group_by={
+        "sex": patients.sex
+    },
+    intervals=months(6).starting_on("2022-01-01"),
+)
+```
+
+The **`name`** argument is just used so we can identify our measure in
+the output. It can be anything you like so long as it contains only
+alphanumeric and underscore characters and starts with a letter.
+
+The **`denominator`** argument defines the condition that patients must
+match to be included in the denominator i.e. it is a [boolean patient
+series](../../reference/language/#BoolPatientSeries). (It is also possible
+to supply an integer here, but we'll cover this later.)
+
+The **`numerator`** arguments defines the _additional_ condition that
+patients must also meet to be included in the numerator. (Again, this
+can also be an integer as we'll discuss later.)
+
+The **`group_by`** argument defines how we would like our results broken
+down. It's optional – leaving it out means that we'll get a single pair
+of numerator/denominator numbers for each time interval – but most
+measures define at least one set of groups. It is supplied as a
+dictionary mapping group names to group definitions.
+
+As we saw in the example above, each group name defined here ends up as
+column in the results. Each group definition should be a "categorical"
+patient series, that is: a patient series which takes only a fixed set
+of values.
```

Comment:
In practice you can actually use anything here because we don't enforce constraints. But it only makes sense to group by series which have this property.